packages/cli/README.md

```diff
@@ -2030,15 +2030,10 @@ const mustBe200 = (response, z, bundle) => {
   return response;
 };
 
-const autoParseJson = (response, z, bundle) => {
-  response.json = z.JSON.parse(response.content);
-  return response;
-};
```

Comment:
This example middleware was just overriding the already set `response.json` so IMO did not have much value as example and fortunately we have another `afterResponse` example.